rockcraft.yaml

```diff
@@ -0,0 +1,36 @@
+name: kratos
+base: bare
+build-base: ubuntu:22.04
+version: "0.13.0"
+summary: Ory Kratos identity server
+description: |
+  Ory Kratos is an Identity and User Management.
+license: Apache-2.0
+platforms:
+  amd64:
+
+services:
+  kratos:
+    override: replace
+    command: kratos serve
+    startup: enabled
+
+parts:
+  kratos:
+    plugin: go
+    build-snaps:
+      - go/1.19/stable
+    build-environment:
+      - GOFLAGS: -ldflags=-w -ldflags=-s
+    source: https://github.com/ory/kratos
+    source-type: git
+    source-tag: v0.13.0
+    prime:
+      - bin/kratos
+      - lib/*/libc.so.6
+      - lib/*/libpthread.so.0
+      - lib/*/ld-linux-x86-64.so.2
```

Comment:
Do we want to be this picky and only include only the shared libraries that are needed?